### PR TITLE
Candles and Prayers

### LINKED ISSUE #5370: Allow players to pray to gods (AKA admins).
## Description
Title. For some reason players are unable to pray to their gods (also known as admins) after admin tools got reworked. This is notable because one of the admin buttons brings up a "Player Prayers" window.

Admin buttons, button with the clasped hands is the "Player Prayers" button.
image

The Player Prayers window, empty of course.
image

### PR DESCRIPTION
**Pull request by AtnerNT**
This PR finally adds functionality to candles allowing them to be lit and burn for light before slowly wittling down.
This PR also adds the ability for players to prayer to admins.
An example of the player prayer menu and the admin prayer menu can be seen below
image

Admin replies to prays will be received in the recipients chat log (Though they will not see which admin sent the message)
The admin UI also has some tools form the player manager to the side for quick access, these tools effect which ever player's prayers are currently being viewed.

Fixes #5370
### Changelog:

CL: [New] Players can now prayer to admins.
CL: [Improvement] Candles can now be lit and burn out over time

### CODE REVIEW

**Comment by AtnerNT** [comment]
Candles last for 10 minutes when on fire

File: UnityProject/Assets/Scripts/Messages/Server/AdminTools/PrayerChatUpdateMessage.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+			AdminChatUpdate update = new AdminChatUpdate();
+			update.messages.Add(chatMessage);
+
+			NetMessage  msg = new NetMessage
+			{
+				JsonData = JsonUtility.ToJson(update),
+				PlayerId = playerId
+			};
+
+			SendToMentors(msg);
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Messages/Server/AdminTools/PrayerChatUpdateMessage.cs]: This correct? not SendToAdmins?

File: UnityProject/Assets/Scripts/Items/Others/Candle.cs [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	public class Candle : NetworkBehaviour, ICheckedInteractable<InventoryApply>, IServerDespawn
+	{
+		private ItemLightControl lightControl;
+
+		[SerializeField]
+		private SpriteHandler spriteHandler = default;
+
+		[SyncVar] public int LifeSpan = 120; //10 minutes
+		[SyncVar] public int DecayStage = 0;
+		[SyncVar] private bool IsOn = false;
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Items/Others/Candle.cs:20]: You don't need to sync var this stuff, the sprite changes will be automatically sent over to clients

File: UnityProject/Assets/Scripts/Items/Others/Candle.cs [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+{
+	[RequireComponent(typeof(ItemLightControl))]
+	public class Candle : NetworkBehaviour, ICheckedInteractable<InventoryApply>, IServerDespawn
+	{
+		private ItemLightControl lightControl;
+
+		[SerializeField]
+		private SpriteHandler spriteHandler = default;
+
+		[SyncVar] public int LifeSpan = 120; //10 minutes
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Items/Others/Candle.cs:18]: Can you add it so if this is -1, then the candle is infinite? or maybe add another bool variable for that?

Would be useful for some maps like the wizard stuff as they have candles.

Comment (AtnerNT) [reply UnityProject/Assets/Scripts/Items/Others/Candle.cs:18]: done

**Review by ThatDan123 — APPROVED** [review]

**Pull request merged**